packages/core/src/dialects/postgres/query.js

```diff
@@ -9,14 +9,16 @@ const { logger } = require('../../utils/logger');
 const debug = logger.debugContext('sql:pg');
 
 export class PostgresQuery extends AbstractQuery {
-  async run(sql, parameters) {
+  async run(sql, parameters, options) {
     const { connection } = this;
 
     if (!_.isEmpty(this.options.searchPath)) {
       sql = this.sequelize.getQueryInterface().queryGenerator.setSearchPath(this.options.searchPath) + sql;
     }
 
-    if (this.sequelize.options.minifyAliases && this.options.includeAliases) {
+    // TODO: https://github.com/sequelize/sequelize/pull/15394#discussion_r1085621226
+    // This can mess up the data since the regexp can replace the original name itself
```

Comment:
It's not blocking to merge this PR, but feel free to open a new issue for this as @ephys mentioned. That way it's more on our radar than a random TODO